pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -526,6 +526,48 @@ type PrometheusSpec struct {
 	// Defines the runtime reloadable configuration of the timeseries database
 	// (TSDB).
 	TSDB TSDBSpec `json:"tsdb,omitempty"`
+	// TracingConfig configures tracing in Prometheus. This is an experimental feature, it may change in any upcoming release in a breaking way.
+	// +optional
+	TracingConfig *PrometheusTracingConfig `json:"tracingConfig,omitempty"`
+}
+
+type PrometheusTracingConfig struct {
+	// Client used to export the traces. Options are "http" or "grpc".
+	//+kubebuilder:validation:Enum=http;grpc
+	//+kubebuilder:default:="grpc"
```

Comment:
(nit) I wouldn't use a default as I prefer to delegate the default  value to Prometheus.